### ISSUE TITLE
Border around tym window

### ISSUE THREAD

**Issue by corelon** [endaaman/tym#112]
![image](https://github.com/endaaman/tym/assets/49140/65fcecd2-7852-41ae-b372-60b3028907d2)
I can see a thick background (tends to become thinner as window size increases) around the tym window. 
I am on Ubuntu 22.04 and manually built the software based on the instructions in the readme.
Is there anything that I did wrong or a setting that I need to change to make this go away?

**Comment by gnsfujiwara** [comment]
I think this is a problem with the font, not the terminal

**Comment by corelon** [comment]
I don't think it is a problem with the font. I did change font with the same effect. When I switch to XOrg or i3/sway it works normally. Could it be a wayland issue?

**Comment by endaaman** [comment]
Maybe stuffs about `color_window_background`. Try `nil`, `""` or `"red"` .
